README.md

```diff
@@ -15,83 +15,186 @@ npm run test
+To get LCP we use the `PageLoadingMetrics` function from `TraceModel`.
 
-it('increment', async () => {
-		const filename = getFileName('Picker');
-		const incrementer = '[class^="Picker_incrementer"]';
+### FPS
 
-		const browser = await puppeteer.launch({headless: true});
-		const page = await browser.newPage();
-		await page.setViewport({
-			width: 1920,
-			height: 1080
-		});
+Frames per second (FPS) is a unit that measures display device performance in video captures and playback and video games. FPS is used to measure frame rate -- the number of images consecutively displayed each second -- and is a common metric used in video capture and playback when discussing video quality. In our case we use it to measure component animation performance.
+To gather average FPS time, we just use the `FPS` function from `TraceModel`.
+For FPS we don't need to specify any components to look for as it will just grab the FPS for the entire page.
 
-		await page.goto('http://localhost:8080/picker');
-		await page.tracing.start({path: filename, screenshots: false});
-		//interactions
-		//...
+### FID
 
-		await page.tracing.stop();
-		await browser.close();
+First Input Delay (FID) measures the time from when a user first interacts with a page (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to begin processing event handlers in response to that interaction.
+To get FID we use the `FID` function from `TraceModel`.
 
-		// FPS
-		const actualFPS = FPS(filename);
-		TestResults.addResult({component: 'Picker', type: 'Frames Per Second', actualValue: actualFPS});
+### CLS
 
-		// Update
-		const actualUpdateTime = Update(filename, 'Changeable');
-		TestResults.addResult({component: 'Picker', type: 'Update', actualValue: actualUpdateTime});
+Cumulative Layout Shift (CLS) is a measure of the largest burst of layout shift scores for every unexpected layout shift that occurs during the entire lifespan of a page.
+A layout shift occurs any time a visible element changes its position from one rendered frame to the next.
+To get CLS we use the `CLS` function from `TraceModel`.
 
-	});
-```
+### Example
 
-### Mount
+Each component is tested repeatedly for both `click` and `keypress` events to measure average FPS. 
+FID and CLS are tested in the same test because they both typically require interactions. We can check the React Devtools to see which component is at the top of a specific component.
+DCL, FCP and LCP are also tested together as they are measured at page load time.
 
-Mounting is very is pretty simple. We just need to find the top component for the component we're trying to test like `Mount(filename, 'Changeable');`. We can check the React Devtools to see which component is at the top of a specific component.
+Results can be found in `testResults` folder and are compared to the optimum values which are stored in global variables declared in `puppeteer.setup` file.

```

Comment:
Can you also mention the thresholds and where they can be modified. (see puppeteer.setup.js)